akka-remote/src/main/resources/reference.conf

```diff
@@ -20,11 +20,11 @@ akka {
 
 
     serialization-bindings {
-      # Since com.google.protobuf.Message does not extend Serializable but
+      # Since akka.protobuf.Message does not extend Serializable but
       # GeneratedMessage does, need to use the more specific one here in order
       # to avoid ambiguity
       "akka.actor.ActorSelectionMessage" = akka-containers
-      "com.google.protobuf.GeneratedMessage" = proto
+      "akka.protobuf.GeneratedMessage" = proto
```

Comment:
`"com.google.protobuf.GeneratedMessage" = proto` may also be included in 2.4, and for 2.3 it must be included